packages/react-button/src/components/Button/ButtonBase.tsx

```diff
@@ -4,8 +4,8 @@ import { compose } from '@fluentui/react-compose';
 import { useButton } from './useButton';
 
 export const ButtonBase = compose<'button', ButtonProps, ButtonProps, {}, {}>(
-  (props, ref, composeOptions) => {
-    const { slots, slotProps } = useButton(props, composeOptions);
+  (props, ref, slotsAndState) => {
```

Comment:
What if the first option is state rather than user props? I don't like the idea of constantly creating new options on every render